hadoop-crypto/src/test/java/com/palantir/crypto2/hadoop/StandaloneEncryptedFileSystemTest.java

```diff
@@ -102,6 +103,49 @@ public void testReadWrite() throws IOException {
         assertThat(rawFs.exists(new Path(path + FileKeyStorageStrategy.EXTENSION))).isTrue();
     }
 
+    @Test
+    public void testDelete() throws IOException {
+        File rootFolder = folder.newFolder();
+
+        Path path1 = writeData(rootFolder);
+        Path path2 = writeData(rootFolder);
+
+        assertThat(efs.exists(path1)).isTrue();
+        assertThat(efs.exists(path2)).isTrue();
+
+        efs.delete(new Path(path1.toString()), false);
+
+        assertThat(efs.exists(path1)).isFalse();
```

Comment:
- should verify that you write to the same location again after deleting from it, I think.

Reply:
don't think that's relevant here? in my mind we've shown that `delete` puts the FileSystem back into a state where a file and it's key material at a given path do not exist. Our `create` tests should verify that we can create a new encrypted file from that state.

Reply:
ok, now that you're actually verifying that, that's fine :)